### PR TITLE
pystun3: add page

### PR DESCRIPTION
**Pull request by marek22k**
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, we generally prefer long options whenever possible.

File: pages/common/pystun3.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Classic STUN client written in python.
+> More information: <https://github.com/talkiq/pystun3>.
+
+- Make a STUN request:
+
+`pystun3`
+
+- Make a STUN request and specify the stun server:
+
+`pystun3 --stun-host {{stun.1und1.de}}`
```
Comment (kbdharun) [inline pages/common/pystun3.md:12]: What is the different between host and stun host?

File: pages/common/pystun3.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Make a STUN request and specify the source port:
+
+`pystun3 --source-port {{7932}}`
```
Comment (kbdharun) [inline pages/common/pystun3.md:16]:
```suggestion
`pystun3 --port {{7932}}`
```

I think you got confused with the argument placeholder and flag

Ref: https://github.com/talkiq/pystun3/blob/681b36ce4812714449dfbf3d2f5004a2f0615240/README.rst?plain=1#L58

File: pages/common/pystun3.md (hunk 1 above)
Comment (marek22k) [reply pages/common/pystun3.md:12]: In what way? In the first example I want to specify the STUN server. I do this with `--stun-host`. In the second example I want to specify the local source port. I do this with `--source-port`. Whereby the second is only conditionally useful if the NAT discovery algorithm of the Classic STUN is used.

Comment (marek22k) [reply pages/common/pystun3.md:12]: Ohh, now I see what you mean. I get the following output:
```
```
usage: pystun3 [-h] [-d] [-H STUN_HOST] [-P STUN_PORT] [-i SOURCE_IP]
               [-p SOURCE_PORT] [--version]

optional arguments:
  -h, --help            show this help message and exit
  -d, --debug           Enable debug logging (default: False)
  -H STUN_HOST, --stun-host STUN_HOST
                        STUN host to use (default: None)
  -P STUN_PORT, --stun-port STUN_PORT
                        STUN host port to use (default: 3478)
  -i SOURCE_IP, --source-ip SOURCE_IP
                        network interface for client (default: 0.0.0.0)
  -p SOURCE_PORT, --source-port SOURCE_PORT
                        port to listen on for client (default: 54320)
  --version             show program's version number and exit
```

Comment (marek22k) [reply pages/common/pystun3.md:12]: It seems like the documentation is wrong: https://github.com/talkiq/pystun3/blob/master/stun/cli.py#L19

Comment (kbdharun) [reply pages/common/pystun3.md:12]: > It seems like the documentation is wrong: https://github.com/talkiq/pystun3/blob/master/stun/cli.py#L19

Yep, I was referring to it. Maybe I will check the code instead.

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**